src/components/Dashboard.js

```diff
@@ -0,0 +1,92 @@
+import React,{useState,useEffect} from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import CreateModal from './CreateModal';
+import Cards from './Cards';
+import Modal  from './Modal';
+import './Dashboard.css';
+
+function Dashboard(){
+
+    const categories = ["Dev-Tasks","Updates-Push","Updates-Pull","Prod-Issue","Code-Review","Meetings"];
```

Comment:
Store any constants in a different file. You can call it `Dashboard.constants.js` and maintain the categories, priorities there.